rules/prefer-string-raw.js

```diff
@@ -64,6 +67,41 @@ const create = context => {
 			},
 		};
 	});
+
+	context.on('TaggedTemplateExpression', node => {
+		const {quasi, tag} = node;
+		const {sourceCode} = context;
+
+		if (!isMemberExpression(tag, {object: 'String', property: 'raw'})) {
+			return;
+		}
+
+		const hasBackslash = quasi.quasis.some(
+			quasi => quasi.value.raw.includes(BACKSLASH),
+		);
+
+		if (hasBackslash) {
+			return;
+		}
+
+		const rawQuasi = sourceCode.getText(quasi);
+		const suggestion = quasi.expressions.length > 0 || /\r?\n/.test(rawQuasi)
+			? rawQuasi
+			: `'${rawQuasi.slice(1, -1).replaceAll('\'', String.raw`\'`)}'`;
+
+		return {
+			node,
```

Comment:
Let's change the report range to point to `String.raw`.

```suggestion
			node: tag,
```